src/responsive-layout.tsx

```diff
@@ -64,16 +65,20 @@ const ResponsiveLayout: React.FC<Props> = ({children, className, fullWidth, data
 
     const sideMargin = isMobile ? MOBILE_SIDE_MARGIN : isTablet ? TABLET_SIDE_MARGIN : null;
 
+    const containerType = isMobile ? 'mobile-column' : isTablet ? 'tablet-column' : 'desktop-wide-column';
```

Comment:
hard to read ternary inside other ternary. 

I'd suggest to reword this to not chain them

Reply:
I just copy pasted the pattern being seen on the above line ;) but you're right, its difficult to read.